.circleci/config.yml

```diff
@@ -58,7 +58,7 @@ jobs:
             - lua_modules
       - run: mkdir -p tmp/junit
       - run: $(make rover) exec make busted
-      - run: JUNIT_OUTPUT_FILE=tmp/junit/prove.xml $(make rover) exec prove --harness=TAP::Harness::JUnit # no carton on centos
+      - run: JUNIT_OUTPUT_FILE=tmp/junit/prove.xml make prove HARNESS=TAP::Harness::JUnit
```

Comment:
Why has this changed from `--harness` to `HARNESS`?

Reply:
OK. Got it in the next commit :+1:

Reply:
becase this needs to pass the harness param to make target 

so the make target has `HARNESS ?= TAP::Harness` and it can be overridden from the outside to a different harness

before it was executing the prove command directly which accepts it as CLI option

Reply:
Why is `HARNESS` defined after the command to run, though?

Reply:
make accepts params like that, or by ENV so it works both before and after (but they can have different semantics, like make can be set to not fetch params from ENV)